### DISCUSSION TITLE
Error building custom avatar style

### DISCUSSION THREAD

**Discussion by misterfunkton** [dicebear/dicebear#174] [Q&A]
Hey there,

I'm trying to make a custom avatar style, and I followed the figma templates linked here on the github to test it out.

When I try to build the file, I do not recieve any errors in the log, but I don't see any new file being output.

`C:\Users\username\Documents\avatar\avatarapp\tinymen>dicebear project build DiceBear.littleguys
(node:425804) [DEP0148] DeprecationWarning: Use of deprecated folder mapping "./" in the "exports" field module resolution of the package at C:\Users\ice cold killa\AppData\Roaming\npm\node_modules\dicebear\node_modules\tslib\package.json.
Update this package.json to use a subpath pattern like "./*".
(Use `node --trace-deprecation ...` to show where the warning was created)
Create dereferences src/schema.ts from schema.json
Create src/options.ts from schema.json
Bundle package with rollup`

Any idea what I could be getting wrong here?

**Comment by FlorianKoerner** [comment]
Your log actually looks good. You can ignore the warning.

Do I understand this correctly? So no "dist" folder has been created for you?

If you like, you can share your Figma file with koernerws@gmail.com. I can then check if I have the same problem.

Windows, Mac or Linux?
